src/javascript/app/pages/user/new_account/digital_options.js

```diff
@@ -0,0 +1,53 @@
+const BinarySocket     = require('../../../base/socket');
+const Client           = require('../../../base/client');
+const getElementById   = require('../../../../_common/common_functions').getElementById;
+const Url              = require('../../../../_common/url');
+const ClientBase       = require('../../../../_common/base/client_base');
+const showLoadingImage = require('../../../../_common/utility').showLoadingImage;
+const State            = require('../../../../_common/storage').State;
+
+const DigitalOptions = (() => {
+
+    let is_virtual,
+        el_welcome_container,
+        upgrade_info,
+        is_uk,
+        is_unwelcome_uk;
+
+    const init = () => {
+        is_virtual            = Client.get('is_virtual');
+        upgrade_info          = ClientBase.getBasicUpgradeInfo();
+        el_welcome_container  = getElementById('welcome_container');
+        is_uk                 = Boolean(Client.get('residence') === 'gb');
+        is_unwelcome_uk       = State.getResponse('get_account_status.status').some(status => status === 'unwelcome') && (/gb/.test(Client.get('residence')));
```

Comment:
if we store `Client.get('residence') === 'gb'` in a variable we can use it in both of these.

Reply:
instead of .some can we also use .includes for the unwelcome?

Reply:
also if this function is used in a few files, might be worth making it a shared function in client.js or similar

Reply:
done all apart from .some change. It doesn't work with includes.